tethys_apps/templatetags/site_settings.py

```diff
@@ -14,23 +14,27 @@
 @register.filter
 @stringfilter
 def load_custom_css(var):
-    if var.startswith("/"):
-        var = var.lstrip("/")
+    if var.startswith(os.sep):
```

Comment:
Does this work on Windows? If a file path is absolute on Windows it will start with something like `C:\` right?